src/devices/Common/Iot/Device/Common/FlagsHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Common
```

Comment:
IMO we should not add helpers like this. `&=~` and `|=` are fine to use rather than this

Reply:
Bitwise and shift operators are not easy to understand when you haven't worked with them before, like myself. The FlagsHelper conveniently hides away such complexity.